src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  height: 408px;
+  padding-top: 32px;
+
+  border: 1px solid $border-color;
+
+  &__container {
+    padding: 0 16px 16px;
+  }
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 1.5;
+    font-weight: 500;
+    color: $main-color;
+
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 1.4;
+    font-weight: 400;
```

Comment:
Let's follow DRY(don't repeat yourself) i see u use it 5 times a row
Can we set it just once and change only where it is needed?